int32 MISO_PORT
int32 MOSI_PORT
int32 CLKA_PORT	
int32 CLKB_PORT 	
int32 CHIP_PORT
float32 LOGIC_LEVEL
int32 SPI_FREQ
int32 SPI_MODE
bool SPI_OPT
bool SPI_GO
int32 NUM_BYTES
int32[] TX_MSG
---
int32[] byteRead
